src/vs/workbench/contrib/chat/common/chatServiceImpl.ts

```diff
@@ -526,6 +582,32 @@ export class ChatService extends Disposable implements IChatService {
 		return this._chatSessionStore.isSessionEmpty(sessionId);
 	}
 
+	getPersistedSessionTitle(sessionId: string): string | undefined {
```

Comment:
We shouldn't have this method, because ChatModel already has a title, and there shouldn't be a second method with different logic to access it